R/utils-get_code_dependency.R

```diff
@@ -0,0 +1,267 @@
+# get_code_dependency ----
+
+#' Return the lines of code (with side-effects) needed to reproduce the object
+#'
+#' @section Notes:
+#' This function assumes that object relationships are established using the `<-`, `=`, or `->` assignment
+#' operators. It does not support other object creation methods like `assign` or `<<-`, nor non-standard-evaluation
+#' methods. To specify relationships between side-effects and objects, you can use the comment tag
+#' `# @linksto object_name` at the end of a line where the side-effect occurs.
+#'
+#' @param code An `expression` with `srcref` attribute or a `character` with the code.
+#' @param names A `character(n)` with object names.
+#'
+#' @return `character` vector of elements of `code` calls that were required to build the side-effects and
+#' influencing objects having and impact on the `object`
```

Comment:
TBD

Reply:
Hey @chlebowa the length of `names` does not have an effect of length of the restulting `character` vector. 548b3b0f39cc8ba1df7a3086a8bf1fc41670c6be

It's gonna be always lenght 1, but it will have more calls collapsed via '\n'